Source/CsDebugScript.DbgEng/DbgEngDll.cs

```diff
@@ -1669,6 +1663,27 @@ private struct KDHELP64
             ulong AddrBase,
             ReadProcessMemoryProc64 ReadMemoryRoutine,
             GetModuleBaseProc64 GetModuleBaseRoutine);
+
+        public IBreakpoint AddBreakpoint(Process process, string expression)
+        {
```

Comment:
These functions are not part of "Native methods" region. Move them outside of the region, preferably before it.

Reply:
Done.